Mathlib/Util/Qq.lean

```diff
@@ -40,4 +40,21 @@ def findLocalDeclWithTypeQ? {u : Level} (sort : Q(Sort u)) : MetaM (Option Q($so
 This is a Qq version of `Lean.Meta.mkDecideProof`. -/
 def mkDecideProofQ (p : Q(Prop)) : MetaM Q($p) := mkDecideProof p
 
+/-- Join a list of elements of type `α` into a container `β`.
+
+Usually `β` is `q(Multiset α)` or `q(Finset α)` or `q(Set α)`.
+Note: this is needed since there aren't `ToExpr` instances on these types, which is to be expected,
+since, e.g, there is in general no canonical way of ordering the elements in a Set/Finset/Multiset.
```

Comment:
Let's just drop this; I think the motivation for the function will be obvious once #24014 lands.
```suggestion
```